core/templates/dev/head/editor/feedback_tab/ThreadDataService.js

```diff
@@ -50,7 +50,8 @@ oppia.factory('threadDataService', [
   };
 
   var _fetchMessages = function(threadId) {
-    $http.get(_THREAD_HANDLER_PREFIX + threadId).success(function(data) {
+    $http.get(_THREAD_HANDLER_PREFIX 
+        + threadId.split(".")[1]).success(function(data) {
```

Comment:
Yep, I think it's fine to do the frontend part as a separate commit, though I'm not sure there are that many changes that would be required in the frontend, so it might be feasible to include after all. I'll leave that up to your discretion, though; I'm fine either way.